docs/src/pages/components/data-grid/accessibility/accessibility.md

```diff
@@ -7,6 +7,4 @@ components: DataGrid, XGrid
 
 <p class="description">The Data Grid has complete accessibility support. For instance, every cell is accessible using the keyboard.</p>
 
-> ⚠️ This feature is only available in v4, head to the [v4 documentation page](https://material-ui.com/components/data-grid/) for more details.
->
-> You can follow [issue #441](https://github.com/mui-org/material-ui-x/issues/441) to be notified once we migrate the data grid from v4 to v5.
+> ⚠️ This feature is now available in v5, head to the [v4 documentation page](https://material-ui.com/components/data-grid/) for more details.
```

Comment:
What do you think about removing notions of the past and instead mention the future?
```suggestion
> ⚠️ The data grid components are supporting v5 and v4. However, no documentation is available for v5 yet. Check the [documentation of v4](https://material-ui.com/components/data-grid/).
```

x the other cases

Reply:
I was also thinking of changing the way it is displayed. That looks like a warning bar. Can we have something like an infobox (different color)?

Reply:
The documentation has no support for different types of messages. So far, we have reproduced a model similar to https://reactjs.org/docs/react-dom-server.html 

It's probably something we could bring cc @mbrookes. I have noticed https://developers.google.com/web/tools/workbox/modules/workbox-window doing such:

<img width="427" alt="Capture d’écran 2021-01-19 à 15 18 36" src="https://user-images.githubusercontent.com/3165635/105046677-afa7ae80-5a69-11eb-8a69-375184c3c78e.png">

<img width="358" alt="Capture d’écran 2021-01-19 à 15 18 40" src="https://user-images.githubusercontent.com/3165635/105046681-b0404500-5a69-11eb-98a5-3372792de245.png">

Reply:
In our case, I think that it should be a warning, the page is useless, it's a placeholder.

Reply:
> It's probably something we could bring

Not sure how we'd distinguish one block-quote from another for the purpose of styling?

Reply:
@mbrookes Yeah, it would likely require custom HTML.